### ISSUE TITLE
[MICROGRANT] Loader

### ISSUE THREAD

**Issue by poolsar42** [ipfs/devgrants#149]
### 1. What is your project? (max 100 words)

Loader is a desktop IPFS client. Think of it as a boilerplate code for electron and go-ipfs.  The main purpose for it - is to integrate this application with [Sytime](https://github.com/ipfs/devgrants/issues/148) to make Sytime offline-first desktop app. 

  https://github.com/HiddenThunder/loader

### 2. How are you planning to improve this project? (max 200 words)

Fix current bugs and that's it.
After that it'll be possible to just get started with ipfs and electron as boilerplate from this app.

### 3. Will the work be Open Source?

Of course.
### 4. If selected, do you agree to complete weekly updates and a grant report upon conclusion?

  Yes.
### 5. Does your proposal comply with our Community Code of Conduct?

Yep!
### 6. Links and submissions

  
* Have you submitted to the IPFS Community Showcase ([instructions](https://github.com/ipfs/community/blob/master/README.md#showcase-your-project))?
  
It's not necessary now, right? I have Sytime included already, but I still think, that this project is separate from Sytime
Sytime is here: https://github.com/ipfs/community/commit/d7dbeb162962d551f95e3f0f7631d1d931802816
* Have you added your project to the [IPFS Ecosystem Directory](https://airtable.com/shrjwvk9pAeAk0Ci7)? 
  Yes
* If your project began at a hackathon, have you submitted it for the relevant Protocol Labs prizes? Include links here if available:
  Nope, it was spontaneous work
* Have you filled out the [Interplanetary Builder Feedback Survey](https://airtable.com/shrDZMizx03jOa4mQ)? 
  Yes
### Additional questions:
* For each team member(s), please list name, email, Github account, and role in the project.
* Emil - poolsar42(github), poolsar.42@gmail.com - lead
* If your project was created as part of an event or hackathon:
* It was not 
  * What was the name of the event? (e.g. ETHGlobal NFTHack, Cal Hacks hello:world, Chainlink, CivHacks, GameDevJ, ETHGlobal Scaling Ethereum)
  * Please link to your hackathon submission, proving you're a team that submitted for a Protocol Labs prize.

**Comment by ErinOCon** [comment]
Hi @poolsar42, thank you for your proposal. We will review this and get back to you, on this thread, with a status update or questions.

**Comment by ErinOCon** [comment]
Hi @poolsar42, thank you for this submission! Generally speaking, it is our process to approve one request at a time for microgrant applicants. Once the Sytime project is complete, please feel welcome to submit a new Issue. 

In the meantime, perhaps this [link](**https://docs.fleek.co/space-daemon/overview/**) can be a helpful resource.

**Comment by poolsar42** [comment]
@ErinOCon yes, it was my concern about it, it's okay.

Essentially loader is just necessary addition to Sytime, but it grew up as different project. So I decided to apply them separately, but I thought that I would be able to explain it in more details

Still, I'm happy with results and you made me much happier
Thank you <3

**Comment by poolsar42** [comment]
Link you sent would've been handful some time ago, but for now I have eveything I need to build

I think

**Comment by ErinOCon** [comment]
Thanks, @poolsar42! Looking forward to seeing your work progress!
